Cargo.toml

```diff
@@ -325,7 +325,11 @@ strip = "none"
 
 # Include debug info in benchmarks too.
 [profile.bench]
-inherits = "profiling"
+opt-level = 3
+lto = "thin"
+debug = "full"
+strip = "none"
+codegen-units = 16
```

Comment:
yeah i think both left and right is benign; it's likely the vscode toml extension is behind cargo on features. if that's annoying enough to change the config then this is cool